Cybersecurity New Topic: Social engineering (#1698)
Codecademy/docs: content/cybersecurity/concepts/social-engineering/social-engineering.md

---
Title: 'Social Engineering'

> Description: 'Social engineering is whne an attacker uses human emotion (usually fear and urgency) to trick the target into performing an action, such as send the attacker money, divulge sensitive customer information, or disclose authentication credentials.'

Reviewer: ```suggestion
Description: 'Social engineering is when an attacker uses human emotion (usually fear and urgency) to trick the target into performing an action, such as sending the attacker money, divulging sensitive customer information, or disclosing authentication credentials.'
```

Subjects:
  - 'Cybersecurity'
Tags:
  - 'Cybersecurity'
  - 'Hacking'
CatalogContent:
  - 'paths/fundamentals-of-cybersecurity'
  - 'introduction-to-personal-digital-security'
---

> **Social Engineering** is when an attacker uses human emotion (usually fear and urgency) to trick the target into performing an action, such as send the attacker money, divulge sensitive customer information, or disclose authentication credentials.

Reviewer: ```suggestion
**Social Engineering** is when an attacker uses human emotion (usually fear and urgency) to trick the target into performing an action, such as sending the attacker money, divulging sensitive customer information, or disclosing authentication credentials.
```

> ## Social Engineering In-Use

Reviewer: Can we exchange this header for what kind of attack this is or name it in the section and link out to: https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack

Author: I don't fully understand this ask. Can you elaborate?

Reviewer: Please disregard this comment!

An individual might receive an email from an unsuspecting user telling them that their account has been compromised and will be deactivated unless they click the link in the email and confirm their credit card details. Once the individual clicks the link in the email, it navigate the individual to a fake website where they can insert their credit card details. 

## How Do Businesses Defend Against Social Engineering Attacks?

> Businesses educate and train employees across the organization. Individuals at all levels should understand not to click on suspicious links or to accept unusual offers, no matter how legitimate the email appears. The following are some quick tips to remember:

Reviewer: ```suggestion
Businesses educate and train employees across the organization. Employees at all levels should understand not to click on suspicious links or to accept unusual offers, despite how legitimate an email appears to be. The following are some quick tips to remember:
```

> - _Think Before Clicking_: Attackers employ a sense of urgency to make you act first and think later in social engineering attacks. If you receive an email with a sense of urgency that seems unusual, it is best to take a moment to check if the source is credible first.
> - _Research The Sources_: Check the domain links to see if they are real. Usually, a typo/spelling error is an indicator that something isn’t right. Hovering your cursor on a link before you actually click on it will reveal the where the link will send you. 
> - _Be Careful Downloading_: If you do not know the sender, it’s best not to open the message. Cybercriminals will often use email attachments to spread viruses and other forms of malware.

Reviewer: ```suggestion

- Think Before Clicking: Attackers employ a sense of urgency to make you act first and think later in social engineering attacks. If you receive an email with a sense of urgency that seems unusual, it is best to take a moment to check if the source is credible first.
- Research The Sources: Check the domain links to see if they are real. Usually, a typo/spelling error is an indicator that something isn’t right. Hovering your cursor on a link before you actually click on it will reveal the where the link will send you. 
- Be Careful Downloading: If you do not know the sender, it’s best not to open the message. Cybercriminals will often use email attachments to spread viruses and other forms of malware.
```

Reviewer: Let's remove pronouns like "you" and "your" and reword accordingly

---
Title: 'Social Engineering'
Description: 'Social engineering is whne an attacker uses human emotion (usually fear and urgency) to trick the target into performing an action, such as send the attacker money, divulge sensitive customer information, or disclose authentication credentials.'
Subjects:
  - 'Cybersecurity'
Tags:

>   - 'Cybersecurity'

Reviewer 2: Is there another tag we can add?

CatalogContent:
  - 'paths/fundamentals-of-cybersecurity'
  - 'introduction-to-personal-digital-security'
---

> **Social Engineering** is whne an attacker uses human emotion (usually fear and urgency) to trick the target into performing an action, such as send the attacker money, divulge sensitive customer information, or disclose authentication credentials.

Reviewer 2: ```suggestion
**Social Engineering** is when an attacker uses human emotion (usually fear and urgency) to trick the target into performing an action, such as send the attacker money, divulge sensitive customer information, or disclose authentication credentials.
```

## Social Engineering In-Use:

An individual might receive an email from an unsuspecting user telling them that their account has been compromised and will be deactivated unless they click the link in the email and confirm their credit card details. Once the individual clicks the link in the email, it navigate the individual to a fake website where they can insert their credit card details. 

## How Do Businesses Defend Against Social Engineering Attacks?

Businesses educate and train employees across the organization. Individuals at all levels should understand not to click on suspicious links or to accept unusual offers, no matter how legitimate the email appears. The following are some quick tips to remember:

> - _Think Before Clicking_: Attackers employ a sense of urgency to make you act first and think later in social engineering attacks. If you receive an email with a sense of urgency that seems unusual, it is best to take a moment to check if the source is credible first.

Reviewer 2: Remove the line breaks between the bullet points.
